rotkehlchen/premium/sync.py

```diff
@@ -196,7 +198,9 @@ def maybe_upload_data_to_server(
             self.last_upload_attempt_ts = ts_now()
             return False, message
 
-        data, our_hash = self.data.compress_and_encrypt_db()
+        current_pool = gevent.get_hub().threadpool
+        greenlet = current_pool.spawn(self.data.compress_and_encrypt_db)
```

Comment:
nothing to lose by doing this right?

```suggestion
        greenlet = gevent.get_hub().threadpool.spawn(self.data.compress_and_encrypt_db)
```